### DISCUSSION TITLE
Github Student Benefits Not yet available

### DISCUSSION THREAD

**Discussion by mohanakrishnan1729** [community/community#148045] [GitHub Education]
### Hi everyone,

Its been 4 days, my education benefits are not activated, what can i do now ?

**Comment by ahmad9059** [comment]
Bro what it is showing pending or verified. if pending wait for few days you will get that if you put the correct details.

**Reply by mohanakrishnan1729** [reply]
it shows pending, 
ok i wait for some more days

**Reply by mohanakrishnan1729** [reply]
yep, 
and happy news I got my education pack,
thank you all ;-)

**Comment by ahmad9059** [comment]
in my case it took 4 days to get that

**Reply by mohanakrishnan1729** [reply]
oh nice, then i wait for it

**Reply by mohanakrishnan1729** [reply]
happy news I got my education pack,
thank you all ;-)

**Comment by uditha2001** [comment]
if it's show as pending then you have to wait few days.

**Reply by mohanakrishnan1729** [reply]
ok

**Reply by mohanakrishnan1729** [reply]
happy news I got my education pack,
thank you all ;-)

**Comment by NS-Sheam** [comment]
No worries. If you submitted the details then you should wait sometime for review. After reviewing your details you can use the student developer pack.

**Reply by mohanakrishnan1729** [reply]
Ok

**Reply by mohanakrishnan1729** [reply]
happy news I got my education pack,
thank you all ;-)

**Reply by mohanakrishnan1729** [reply]
no, i just wait for sometimes, 
then it automatically successful ,
i didnt do anything,

**Reply by NS-Sheam** [reply]
Congratulations. Enjoy the developer pack.

**Comment by davevad93** [comment]
Hi @mohanakrishnan1729 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @mohanakrishnan1729 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
